src/admin/content-block/helpers/generators/accordions.ts

```diff
@@ -0,0 +1,45 @@
+import i18n from '../../../../shared/translations/i18n';
+
+import {
+	AccordionsBlockComponentState,
+	ContentBlockBackgroundColor,
+	ContentBlockConfig,
+	ContentBlockEditor,
+	ContentBlockType,
+	DefaultContentBlockState,
+} from '../../content-block.types';
+import { CONTENT_BLOCK_FIELD_DEFAULTS, FORM_STATE_DEFAULTS, TEXT_FIELD } from './defaults';
+
+const EMPTY_ACCORDION: AccordionsBlockComponentState = {
+	title: '',
+	content: '',
+};
+
+export const INITIAL_ACCORDIONS_BLOCK_COMPONENT_STATES = (): AccordionsBlockComponentState[] => [
+	EMPTY_ACCORDION,
+];
+
+export const INITIAL_ACCORDIONS_BLOCK_STATE = (position: number): DefaultContentBlockState =>
+	FORM_STATE_DEFAULTS(ContentBlockBackgroundColor.White, ContentBlockType.Accordions, position);
+
+export const ACCORDIONS_BLOCK_CONFIG = (position: number = 0): ContentBlockConfig => ({
+	name: i18n.t('Accordeons'),
+	components: {
+		name: i18n.t('Accordeon'),
+		limits: {
+			min: 1,
+			max: 8,
```

Comment:
wie bepaald deze limiet?

Reply:
Wij voor de moment, maar de accordions gaan anders in de config zitten afhankelijk van hoe de FAQ items gaan opgebouwd worden. Dit gaat vervangen worden door een ContentPicker idealiter.